src/components/Facets/Scalar/ScalarFacet.tsx

```diff
@@ -22,11 +23,30 @@ export const ScalarFacet: FunctionComponent<ScalarFacetProps> = ({
 }) => {
   const { isSelected, onChange } = useScalarFacet(filterData);
 
+  const collapsedBuckets = useMemo(() => {
```

Comment:
Collapse multiple cateogry buckets that share the sme name. This occurs when viewing grouped products that all have a common parent category name. Filtering on a category name will be applied across all buckets that use that category name. Similarly, removing the category name form the selected filters will remove ALL associated bucket values that use the same category name.